react/package.json

```diff
@@ -20,7 +20,15 @@
     "@types/jest": "^24.0.18",
     "@vtex/test-tools": "^0.1.2",
     "@vtex/tsconfig": "^0.1.1",
-    "apollo-client": "^2.5.1"
+    "apollo-client": "^2.5.1",
+    "vtex.category-menu": "https://cristi--elefant.myvtex.com/_v/private/typings/linked/v1/vtex.category-menu@2.16.1+build1627629726/public/@types/vtex.category-menu",
```

Comment:
This line can't be the type of your workspace. Unlink the app from your current workspace, then run `vtex setup --typings`